tests/test_purchase_label.py

```diff
@@ -100,8 +104,12 @@ def test_purchase_label_using_reference_ids(self, api: shippo.Shippo):
         assert shipment.address_from.object_id == address_from.object_id
         assert shipment.address_return.object_id == address_from.object_id
 
+        order_object_id = get_first_order_object_id(api)
+        assert order_object_id is not None
+        assert isinstance(order_object_id, str)
```

Comment:
same here...is it possible to assert matching rather than just existence and type assertion?

Reply:
Yeah, we really should re-factor all the tests in this repo b/c none of them verify beyond existence and type.
I am exploring automated test generation currently and trying to avoid scope creep on resolving this issue but it is something we should do. I'll open an additional ticket to tackle this larger effort.